src/cow-react/modules/limitOrders/containers/TradeButtons/limitOrdersTradeButtonsMap.tsx

```diff
@@ -63,7 +63,7 @@ export function SwapButton({
 }
 
 const quoteErrorTexts: { [key in GpQuoteErrorCodes]: string } = {
-  [GpQuoteErrorCodes.UNHANDLED_ERROR]: 'Unhandled error',
+  [GpQuoteErrorCodes.UNHANDLED_ERROR]: 'Error loading price. Try again later.',
```

Comment:
While this fixes the immediate problem, I have questions:

1. Are all unhandled errors price loading errors?
2. Can we handle them somehow as price loading errors don't end up as unhandled errors at this point?

Reply:
1. All these errors are from `quote` requests, so, all of them are related to price loading
2. UNHANDLED_ERROR it's any `5xx` HTTP errors